src/fornberg.jl

```diff
@@ -55,23 +55,24 @@ immutable LinearOperator{T<:Real,S<:SVector,LBC,RBC} <: AbstractLinearOperator{T
             low_boundary_coefs,
             high_boundary_coefs,
             boundary_fn
-        )
+            )
     end
     (::Type{LinearOperator{T}}){T<:Real}(dorder::Int, aorder::Int, dx::T, dim::Int, LBC::Symbol, RBC::Symbol; bndry_fn=(0.0,0.0)) =
-    LinearOperator{T, SVector{dorder+aorder-1+(dorder+aorder)%2,T}, LBC, RBC}(dorder, aorder, dx, dim, bndry_fn)
+        LinearOperator{T, SVector{dorder+aorder-1+(dorder+aorder)%2,T}, LBC, RBC}(dorder, aorder, dx, dim, bndry_fn)
 end
 
 
 function initialize_boundaries!{T}(low_boundary_coefs,high_boundary_coefs,
-                                   derivative_order,grid_step::T,boundary_length,
-                                   boundary_point_count,LBC,RBC)
+                                   derivative_order, approximation_order, grid_step::T,boundary_length,
+                                   boundary_point_count,LBC,RBC, stencil_coefs)
     high_temp            = zeros(T,boundary_length)
     flag                 = derivative_order*boundary_point_count%2
     aorder               = boundary_length - 1
```

Comment:
`aorder` is same as `approximation_order`, you can use this directly instead of getting it from function definition.